.github/workflows/ci.yml

```diff
@@ -3,6 +3,20 @@ name: CI
 on: [pull_request]
 
 jobs:
+  presubmit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python environment
+      uses: actions/setup-python@v1.1.1
+      with:
+        python-version: 3.7
+    - name: Build benchmarks
```

Comment:
This name should "Run presubmit checks"
Maybe even rename one below from "Build benchmark" to Build (that fuzzer name var) with all benchmarks ?

Reply:
Done. Ill try to figure out renaming the one below later.